### ISSUE TITLE
[1.18] Chickens still spawn even if they are disabled

### ISSUE THREAD

**Issue by frifre10** [CreeperHost/Chickens#21]
Hello, for my modpack I wanted to add the chickens but when checking everything, the chickens continue to spawn despite having put "NONE" in "spawn_type". Does anyone have a solution, I'm playing in 1.18

```
  "chickens:netherite_scrap_chicken": {
    "name": "netherite_scrap_chicken",
    "is_enabled": true,
    "lay_coefficient": 1.0,
    "lay_item": {
      "itemID": "minecraft:netherite_scrap"
    },
    "drop_item": {
      "itemID": "minecraft:netherite_scrap"
    },
    "spawn_type": "NONE",
    "parent_1": "", 
    "parent_2": ""
  },
```

**Comment by gigabit101** [comment]
Fixed in 1.0.20, Now available on CurseForge

**Comment by TheOutcast200** [comment]
hey sorry wanted to ask you how to add chickens in the mod im on 1.19.2 if the code above works im ok with using that the only prob idk how to work coding or anything
